### DISCUSSION TITLE
How to Build apk with implementation 'com.github.rubenlagus:TelegramBots:version' ?

### DISCUSSION THREAD

**Discussion by skynight137** [rubenlagus/TelegramBots#1251] [Q&A] [answered]
is it possible to build apk using implementation 'com.github.rubenlagus:TelegramBots:6.7.0' ?
 .  I'm trying to build an apk by adding this implementation to my apk, but when I process the gradle, an error appears.  FAILURE: Build failed with an exception.
 * What went wrong:
 Execution failed for task ':app:checkReleaseDuplicateClasses'.
 > A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
    > Duplicate class org.aopalliance.aop.Advice found in modules jetified-aopalliance-repackaged-2.6.1 (org.glassfish.hk2.external:aopalliance-repackaged:2.6.1) and jetified-spring-aop-5.3.27 (org.springframework:spring-aop:5.3.27)...
blablabla...
 ....someone please.....

**Comment by rubenlagus** [comment, marked as answer]
The library is not built for Android by default but would suggest to use the new version 7.X.X since it contains many improvements.
